### DISCUSSION TITLE
nightly build

### DISCUSSION THREAD

**Discussion by yourfriendisaac** [corbindavenport/cobalt#15] [General]
virus detected
![image](https://user-images.githubusercontent.com/7629717/158075420-9b114e08-b209-4593-b562-705a5f94ca1f.png)

**Comment by corbindavenport** [comment]
Not really sure what's going on here, Windows on my PC is flagging it as "Trojan:Script/Conteban.A!ml", but there's no information about what it is or what in the ISO could be causing it: https://www.microsoft.com/en-us/wdsi/threats/malware-encyclopedia-description?name=Trojan%3aScript%2fConteban.A!ml&threatid=2147735508

On Windows 10, you can override this by opening **Windows Security > Protection history > Threat blocked > Actions > Allow**, and then downloading again. I'll have time to look into this more later, but I can't imagine there's a trojan in the DOS software unless somehow iBiblio was compromised.

**Comment by corbindavenport** [comment]
I think this is fixed now, the latest nightly build isn't being flagged by Defender anymore.
